controller/engine_image_controller.go

```diff
@@ -762,7 +762,8 @@ func (ic *EngineImageController) createEngineImageDaemonSetSpec(ei *longhorn.Eng
 									},
 								},
 								InitialDelaySeconds: 5,
```

Comment:
I did not change this to use `datastore.PodProbeInitialDelay` since the value is 5 bigger than 3.